src/libsync/syncengine.cpp

```diff
@@ -82,8 +83,8 @@ SyncEngine::SyncEngine(AccountPtr account, const QString& localPath,
     qRegisterMetaType<SyncFileItemVector>("SyncFileItemVector");
     qRegisterMetaType<SyncFileItem::Direction>("SyncFileItem::Direction");
 
-    // Everything in the SyncEngine expects a trailing slash for the localPath.
-    Q_ASSERT(localPath.endsWith(QLatin1Char('/')));
+    WASSERT(localPath.endsWith(QLatin1Char('/')),
```

Comment:
I'd keep this one a Q_ASSERT, it doesn't depend on outside data, and you'll get something that break in auto tests if this is hit anyway.